src/BUILD

```diff
@@ -572,6 +572,25 @@ cc_library(
     deps = ["//src:shared_lib"]
 )
 
+cc_binary(
```

Comment:
Is current scope limited to running in inside container based on build image of OVMS?